### DISCUSSION TITLE
Training new entity without forgetting old ones

### DISCUSSION THREAD

**Discussion by itamar12345** [explosion/spaCy#7922] [Help: Best practices]
Hello,
When I'm training a new entity for NER pipline in spacy, I also need to use training data of the original entities in order to not harm the model too much. 
Is it possible to add a new entity and train it solely without changing (harming) the other entities accuracy?

**Comment by svlandeg** [comment]
No, that's not possible, you really do need to add in training data of the other entities.
If you don't have access to gold-standard training data of those other entities, you can also use predictions of the original model and feed those in among your new entities, to ensure that the model "remembers" how to predict the old ones.

**Reply by michelemarzollo** [reply]
Hello,
I was wondering if training a separate NER model only for the new entity types wouldn't be better. Of course it would affect performance to run two different models, but, in terms of accuracy, do you think it is something that could be worth it?
Thank you for any suggestions!

**Reply by polm** [reply]
Hard to say how well it would work without getting into the details of your data, but training a second model is almost certainly worth trying. Without knowing more about your use case, I would train it with some default settings and try putting it before and after the existing NER and see how it does. 

Also going to link to your other thread https://github.com/explosion/spaCy/discussions/8288 since I guess that's related.

**Reply by michelemarzollo** [reply]
Thank you!
